docs/quickstart/cross-chain-transfers.md

```diff
@@ -0,0 +1,176 @@
+# Transfer AVAX Tokens Between Chains
+
+## Introduction
+
+This article shows how to transfer AVAX tokens programmatically between any two chains of the Primary Network.
+
+If you are looking for how to transfter AVAX tokens using the web wallet, please check out [this article](https://support.avax.network/en/articles/6169872-how-to-make-a-cross-chain-transfer-in-the-avalanche-wallet).
+
+## Requirements
+
+You've completed [Run an Avalanche Node](../nodes/build/run-avalanche-node-manually.md) and are familiar with [Avalanche's architecture](../overview/getting-started/avalanche-platform.md). You are familiar with the [AvalancheJS](https://github.com/ava-labs/AvalancheJS) repo.
+
+To use AvalancheJS, you can clone the repo or add it to an existing project by running the following command:
+
+```zsh
+$ yarn add --dev avalanche
+```
+For this tutorial we will use a cli to run the example scripts directly from an AvalancheJS directory.
+
+
+In order to send AVAX, you need to have some AVAX! You can get real AVAX by buying it on an exchange, or you can get testnet AVAX from the [AVAX Test Faucet](https://faucet.avax.network), which is a free and easy way to get to play around with Avalanche.
+
+
+## Transferring AVAX using AvalancheJS
+
+The easiest way to transfer AVAX between chains is to use [AvalancheJS](https://github.com/ava-labs/AvalancheJS) which is a programmatic way to access and move AVAX.
```

Comment:
Add that we think using the AvalancheGo's keystore is an anti-pattern because it requires you to keep your keys on a full node which makes them a target for malicious hackers. Instead the recommended way is to use AvalancheJS which allows you to create and sign transactions locally.

Reply:
Following up this comments, please show in the code where to put the private key.

Reply:
https://github.com/ava-labs/avalanche-docs/blob/ebc232b5fa0f117a69328a08c3a7142613b6c13a/docs/quickstart/cross-chain-transfers.md?plain=1#L51